garrysmod/lua/includes/extensions/util.lua

```diff
@@ -408,3 +408,15 @@ function util.IsBinaryModuleInstalled( name )
 
 	return false
 end
+
+function util.In( thing, ... )
+    for i = 1, select( '#', ... ) do

```

Comment:
nit: I think it was Spar who demonstrated that localizing your for limit makes it marginally faster.

Something like:
```suggestion
    local limit = select( "#", ... )

    for i = 1, limit do
```

_might_ be a nice microoptimization

Reply:
I don't see how this is an improvement since the loop limit is only evaluated once.
![image](https://github.com/Facepunch/garrysmod/assets/4593679/d7eb7dd2-31cb-4690-b5a3-acc4e800535e)

Reply:
> I don't see how this is an improvement since the loop limit is only evaluated once.

Does it be have like that in GLua too?

Reply:
Yes